includes/class-sensei-course.php

```diff
@@ -224,6 +224,14 @@ public function add_showcase_courses_upsell() {
 	 * @internal
 	 */
 	public function showcase_courses_screen() {
+		// Get the price of Pro. Return if it's not available.
+		$sensei_pro_product = Sensei_Extensions::instance()->get_extension( 'sensei-pro' );
```

Comment:
We could have a constant for Sensei Pro, no? Maybe on `Sensei_Extensions` itself. Just a small nitpick. :)

Reply:
👍🏼  Changed in b669feb819e084fb1301d8e815fdeece8df8288f